Octokit.Reactive/Clients/IObservableOrganizationsClient.cs

```diff
@@ -16,6 +16,12 @@ public interface IObservableOrganizationsClient
         IObservableTeamsClient Team { get; }
 
         /// <summary>
+        /// A client for GitHub's Organization Hooks API.
+        /// </summary>
+        /// <remarks>See <a href="http://developer.github.com/v3/orgs/hooks/">Hooks API documentation</a> for more information.</remarks>
+        IObservableOrganizationHooksClient Hooks { get; }
```

Comment:
This should be in the singular - `Hook` to be consistent with the other sub-client properties